### PR TITLE
Feature profile details

### PR DESCRIPTION
**Pull request by c-s-ale**
### What this PR does (required):
Adds a profile page, along with a card that can be used to book the sitter.

### Screenshots / Videos (required):
image

### Any information needed to test this feature (required):
This feature is hardcoded, and cannot be reached with the current routing.

### Any issues with the current functionality (optional):
Disconnected from backend

### CODE REVIEW

File: client/src/components/BookSitter/BookSitter.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import house from '../../Images/house.jpg';
+import useStyles from './useStyles';
+
+const BookSitter = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+
+  return (
+    <Card className={classes.mainCard} elevation={5}>
+      <Paper elevation={3} className={classes.paper}>
```
Comment (rajivtitus) [inline client/src/components/BookSitter/BookSitter.tsx:24]: I think we can exclude the Paper component over here? As cards have elevation and box-shadows by default.

File: client/src/components/SitterProfile/SitterProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    {
+      img: cuteDogThree,
+      title: 'Even more dog',
+    },
+  ];
+
+  return (
+    <Card className={classes.mainCard} elevation={5}>
+      <CardMedia className={classes.cardMedia} image={house} />
+      <Paper className={classes.mainPaper} elevation={3}>
```
Comment (rajivtitus) [inline client/src/components/SitterProfile/SitterProfile.tsx:51]: Ditto as above.

File: client/src/pages/SitterProfile/SitterProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import Grid from '@material-ui/core/Grid';
+import useStyles from './useStyles';
+import SitterProfile from '../../components/SitterProfile/SitterProfile';
+import BookSitter from '../../components/BookSitter/BookSitter';
+
+export default function Register(): JSX.Element {
+  const classes = useStyles();
+  //const { updateLoginContext } = useAuth();
```
Comment (rajivtitus) [inline client/src/pages/SitterProfile/SitterProfile.tsx:8]: remove commented code

File: client/src/components/SitterProfile/SitterProfile.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+          </Grid>
+          <Grid item className={classes.gridItem}>
+            <Typography className={classes.location}>
+              <LocationOnIcon color="primary"></LocationOnIcon>Toronto, Ontario
+            </Typography>
+          </Grid>
+        </Grid>
+
+        <CardContent className={classes.cardContent}>
+          <Grid container direction="column">
```
Comment (rajivtitus) [inline client/src/components/SitterProfile/SitterProfile.tsx:70]: I think we can avoid using a Grid over here, the Typography by itself should give us the same result. Unless there's any other reason for using a Grid?

**Review by rajivtitus — APPROVED** [review]
Nice work! UI looks really good.

Please review comments before merging.

File: client/src/components/BookSitter/BookSitter.tsx (hunk 1 above)
Comment (c-s-ale) [reply client/src/components/BookSitter/BookSitter.tsx:24]: Yes, I removed it without any impact - thank you!

File: client/src/components/SitterProfile/SitterProfile.tsx (hunk 2 above)
Comment (c-s-ale) [reply client/src/components/SitterProfile/SitterProfile.tsx:51]: Removed the Paper element with no issues.

File: client/src/components/SitterProfile/SitterProfile.tsx (hunk 4 above)
Comment (c-s-ale) [reply client/src/components/SitterProfile/SitterProfile.tsx:70]: I removed the grid element and tested the page and it seems to be the same total outcome - thank you! I didn't think Typography would shrink the same way as the Grid item would, but it absolutely does!

**Comment by c-s-ale** [comment]
image

Revision based on feeback!

**Comment by rajivtitus** [comment]
> image
> 
> Revision based on feeback!

Looks great! Please feel free to merge when possible. Thank you

**Pull request merged**